docs/02-app/01-building-your-application/11-upgrading/02-version-15.mdx

```diff
@@ -3,43 +3,441 @@ title: Version 15
 description: Upgrade your Next.js Application from Version 14 to 15.
 ---
 
-{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
-
 ## Upgrading from 14 to 15
 
-To update to Next.js version 15, run the following command using your preferred package manager:
-
-```bash filename="Terminal"
-npm i next@rc react@rc react-dom@rc eslint-config-next@rc
-```
+To update to Next.js version 15, you can use the `upgrade` codemod:
 
 ```bash filename="Terminal"
-yarn add next@rc react@rc react-dom@rc eslint-config-next@rc
+npx @next/codemod@canary upgrade
```

Comment:
We should adjust this once we have an RC available, but for now this is relevant for folks trying it out on canary.

Reply:
I feel good to leave `@canary` here, if we have further updates we also want users to get the latest script

Reply:
FYI the upgrade command is structured in a way that it defaults the designated Next.js dist-tag to the dist-tag the codemod was run in. 

So if you want to upgrade to `canary`, you run `@next/codemod@canary`. If you want to update to `latest`, you use `@next/codemod@latest`. 

You can still choose a different dist-tag but the recommendation should be to match codemod dist-tag and Next.js dist-tag since that requires less user input.